Add July meeting notes (#56)
python/docs-community: docs/monthly-meeting/2022-07.md

* Diátaxis workshop has a date & signup form: 
  - [Announcement post](https://discuss.python.org/t/17075)
  - [Registration form](https://forms.gle/xQSZJ2Yf4Jjd27wf6)
  - Dates: 16 Aug (pt. 1) & 18 Aug (pt. 2), both 4pm-6pm UTC

## Agenda items

### Past Action Items

- [ ] CAM: Carol (and CAM?) will be updating the readme and guide at the sprints.
  - Update: CAM didn't see any reviews or feedback on his PRs. (He did take some time off from Python work.)
  - CAM: Petr ended up more or less doing this first, so I think we can check this off
- [x] Adam: Creating a 'docs-approved' (wording to be agreed) label for CPython.
  - https://github.com/python/core-workflow/issues/458
  - Created as the `@python/proofreaders` team
- [x] Petr: Start the Docs WG → Editorial Board rename
  - https://github.com/python/docs-community/pull/55
- [x] Petr: Solicit agenda items on Discourse for the next/future meetings
  - https://discuss.python.org/t/16317
- [x] Petr: Create a `proofreaders` team (Initially: CAM, Adam, Pradyun, Hugo), post about this team on Discourse for a wider audience
  - https://discuss.python.org/t/16319

> - [x] Ezio: Add a “what's new” section to the devguide (key/important changes)
>   - https://github.com/python/devguide/issues/885
>   - Should we use blurb?

Reviewer: ```suggestion
- [ ] Ezio: Add a “what's new” section to the devguide (key/important changes)
  - https://github.com/python/devguide/issues/885
  - Should we use blurb?
    - start simple, then move to blurb if needed
```
The issue has been created, but it hasn't been fixed yet.

### Python Project Documentation

*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, &c.*

* Writing Diataxis guidelines
  * how to approach diataxification
    * internal reorg, doc splitting, etc.
  * making sections recognizable for readers and writers
    * explicit boxes/directives
    * naming conventions for sections
    * naming conventions for anchors
      * (i.e. ``.. _<module>-guide:``; ``.. _<module>-reference:``) 
      * global summary page/table
  * bunch-of-functions modules vs ~frameworks
  * page structure
    * initial paragraph with links vs ToC vs list of sections
    * howtos section organization (titles/etc) / examples vs howtos

* (Not) breaking URLs
  * How much do we care?
    - we should preserve reference links as much as possible. Tutorial/Guide/etc links we can try and preserve the path to the right location through preserving anchors (perhaps in the footer)
  * Automation to prevent breakage?
    * Apply data: Is it possible to look at web server logs to see which fragments are most often followed by incoming visitors?
      * no, anchors aren't in server logs (unless we use JS tracking)

>     * [ ] Try building automation using the .inv file

Reviewer 2: Recommend you take a look at my [sphobjinv](https://github.com/bskinn/sphobjinv) as possible machinery for this. Happy to field any questions.

* Reorganize the devguide in directories
  * https://github.com/python/devguide/pull/901
  * Squash merge or rebase?
    * rebase

* Typing PEP migration

#### Workflow

* Should we apply the `skip news` label to documentation PRs?
  * [Discussion in Bedevere's tracker](https://github.com/python/bedevere/issues/457).
  * PR: https://github.com/python/bedevere/pull/485

>   * Labels semantincs on PRs vs issues (`stdlib`/`tests`/`docs`/etc.)
>   * not blocked

Reviewer: ```suggestion
  * Labels semantics on PRs vs issues (`stdlib`/`tests`/`docs`/etc.)
  * not blocked
```

# Documentation Community Team Meeting (July 2022)

- **Date:** 2022-07-11

> - **Time:** [21:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-07-??/21:00/Docs%20Meeting)-->

Reviewer 3: ```suggestion
- **Time:** [19:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-07-??/21:00/Docs%20Meeting)-->
```

Am I remember wrong, or wasn't it at 19:00 UTC, instead of our normal 20:00 UTC to account for DST/summer time in most time zones?

Reviewer 4: It was at 21:00 UTC, midnight my time... 19:00 UTC would be better for me :)

Reviewer 3: Ah, right, I miscalculated again...my fault. I'm fine with moving it earlier, but we would want to do another WhenIsGood/Doodle/etc. poll again to capture the best time for everybody.

> - **Time:** [21:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-07-??/21:00/Docs%20Meeting)-->

Reviewer 4: ```suggestion
- **Time:** [21:00 UTC]<!--(https://arewemeetingyet.com/UTC/2022-07-11/21:00/Docs%20Meeting)-->
```
